src/GeositeFramework/js/Map.js

```diff
@@ -216,22 +228,15 @@ require(['use!Geosite',
                 x.xmin, x.ymin, x.xmax, x.ymax,
                 new SpatialReference({ wkid: x.spatialReference.wkid })
             );
-        view.esriMap.setExtent(extent);
+        view.esriMapView.extent = extent;
     }
 
     function saveExtent(view) {
-        view.model.set('extent', view.esriMap.extent);
+        view.model.set('extent', view.esriMapView.extent);
     }
 
     function selectBasemap(view) {
-        // Hide the current basemap layer
-        if (view.currentBasemapLayer !== undefined) {
-            view.currentBasemapLayer.hide();
-        }
-        // Show the new basemap layer (at index 0)
-        view.currentBasemapLayer = view.model.getSelectedBasemapLayer(view.esriMap);
-        view.currentBasemapLayer.show();
-        view.esriMap.reorderLayer(view.currentBasemapLayer, 0);
+        view.model.getSelectedBasemapLayer(view.esriMap);
```

Comment:
Similar to the comment on `getSelectedBasemapLayer` above, I'm going to restore the cut code but comment it out with a note explaining that we'll need to update how we configure the basemap layers initially and how we toggle them. It's correct, however, that this is making use of a side-effect to work. 

I think we'd probably prefer that it *not* work like that, so I'll leave the method name the same for now